README.md

```diff
@@ -6,8 +6,13 @@
 
 1. Clone the repo
 2. Run `docker-compose up -d` in the project directory
-3. Copy `.env.example` as `.env` in `projectread/` and add the environment variables
-4. Go to `localhost:8000` to make sure everything works
+3. Copy `.env.example` as `.env` in `projectread/`
+4. Set up your local vault client by following the sections *Install the Vault Client* and *Point to our Vault server* in these [docs](https://www.notion.so/uwblueprintexecs/Secret-Management-2d5b59ef0987415e93ec951ce05bf03e#86337406f266493d990911901480f435)
```

Comment:
I also had to do the "authenticate" step to log in using my PAT! would you mind adding that?